src/refspec.c

```diff
@@ -160,6 +160,35 @@ void git_refspec__free(git_refspec *refspec)
 	memset(refspec, 0x0, sizeof(git_refspec));
 }
 
+int git_refspec_parse(git_refspec **refspec, const char *input, int is_fetch) {
+	git_refspec *new_refspec;
+	if (refspec == NULL || input == NULL) {
+		return -1;
+	}
+
+	is_fetch = !!is_fetch;
```

Comment:
I'd move that inside the `git_refspec__parse` call.

Reply:
done